src/prefect/profiles.toml

```diff
@@ -1,3 +1,20 @@
+# This is a template for profile configuration for Prefect.
+# You can modify these profiles or create new ones to suit their needs.
+
 active = "default"
 
-[profiles.default]
\ No newline at end of file
+[profiles.default]
```

Comment:
I think we can remove the 'default' profile from the starting set of profiles as part of this and make 'ephemeral' the default.